lib/resources/example_json/dstu2_examples_contract.rb

```diff
@@ -498,58 +463,51 @@ module Resources
                     "text": "Federated Person Principal"
                   },
                   "system": "urn:oid:2.16.840.1.113883.3.13.6",
-                  "value": "URN:CERNER:IDENTITY-FEDERATION:REALM:AC193C1A-9763-45E1-9FA2-C1C3AA4BA16D:PRINCIPAL:R53HZ5ZL8ZX",
+                  "value": "NEW-FEDPRINCIPAL-ALIAS-001-MAX-VALIDATION",
```

Comment:
Oops this is still leftover from our deployment verification.

I just patched in a new value that isn't so fake, can you update to "URN:CERNER:IDENTITY-FEDERATION:REALM:6A2A9D0D-C19E-4B52-BB97-22732FDAA8D6:PRINCIPAL:A50B81F9AD"